vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -498,7 +490,8 @@ def __init__(self, quant_config: Fp8Config):
                     "DeepGemm not supported on the current platform.")
 
         # Check for CutlassBlockScaledGroupedGemm support.
-        self.allow_cutlass_block_scaled_grouped_gemm = False
+        self.allow_cutlass_block_scaled_grouped_gemm_sm100 = False
+        self.allow_cutlass_block_scaled_grouped_gemm_sm90 = False
```

Comment:
Can we keep just `allow_cutlass_block_scaled_grouped_gemm` and only do the dispatching in the kernel? We should avoid spreading complexity if possible